hypervisor/common/ptdev.c

```diff
@@ -259,7 +265,7 @@ uint32_t ptirq_get_intr_data(const struct acrn_vm *target_vm, uint64_t *buffer,
 
 	for (i = 0U; i < CONFIG_MAX_PT_IRQ_ENTRIES; i++) {
 		entry = &ptirq_entries[i];
-		if (!is_entry_active(entry)) {
+		if (!is_entry_active(entry) || (entry->allocated_pirq >= NR_IRQS)) {
```

Comment:
should be  != IRQ_INVALID? need to match with the above code, and don;t assume IRQ_INVALID is larger is larger or smaller than NR_IRQs

Reply:
> should be != IRQ_INVALID? need to match with the above code, and don;t assume IRQ_INVALID is larger is larger or smaller than NR_IRQs

yes, fixed.